TappNetwork/home-visiting-app-web
# Cu 2mun4em supervisor detail

# Details
This PR is for front and back end changes adding Supervisor Profile Pages accessible from the visitor profile pages.

# Steps to test changes

1. Pull down both branches
2. Login to front end and select a visitor from the visitor directory
3. Click the link to their supervisor's profile underneath the profile picture
4. Supervisor profile should load.

# Issues Addressed

[2mun4em](https://app.clickup.com/t/2mun4em)
